### ISSUE TITLE
add wav pack support

### ISSUE THREAD

**Issue by Reaper10** [swesterfeld/beast#13]
http://wavpack.com/

**Comment by swesterfeld** [comment]
Yes, we could add support for wavpack. As we do have support for reading flac (also lossless), for me this doesn't look like a high priority issue.
